README.es.md

```diff
@@ -0,0 +1,200 @@
+* `interface Foo { new(): Foo; }`:
+    Este define el tipo de objeto que esten nuevos. Probablemente quieras `declare class Foo { constructor(); }`.
+* `const Class: { new(): IClass; }`:
+    Prefiere usar una declaración de clase `class Class { constructor(); }` En vez de una nueva constante.
+* `getMeAT<T>(): T`:
+    Si un tipo de parámetro no aparece en los tipos de ningún parámetro, No tienes una función generica,solo tienes una afirmación de tipo disfrazada.
+     Prefiera utilizar una afirmación de tipo real, p.ej. `getMeAT() as number`.
+    Un ejemplo donde un tipo de parámetro es aceptable: `function id<T>(value: T): T;`.
+    Un ejemplo donde no es aceptable: `function parseJson<T>(json: string): T;`.
+    Una excepción: `new Map<string, number>()` está bien.
+* Utilizando los types `Function` y `Object` casi nunca es una buena idea. En 99% de los casos es posible especificar un tipo más especifico. Los ejemplos son `(x: number) => number` for [functions](http://www.typescriptlang.org/docs/handbook/functions.html#function-types) and `{ x: number, y: number }` for objects. If there is no certainty at all about the type, [`any`](http://www.typescriptlang.org/docs/handbook/basic-types.html#any) is the right choice, not `Object`. If the only known fact about the type is that it's some object, use the type [`object`](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-2.html#object-type), not `Object` or `{ [key: string]: any }`.
+* `var foo: string | any`:
+    Cuando es usado `any` En un tipo de unión, El tipo resultante todavía es `any`. Así que mientras la porción `string` de este tipo de anotación puede _verse_ útil, de hecho, no ofrece ningún typechecking adicional más que un simple `any`.
+    Dependiendo de la intención, una alternativa aceptable puede ser `any`, `string`, o `string | object`.
+
+
+#### Remover un paquete
+
+Cuando un paquete [bundles](http://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html) sus propios tipos, Estos tipos deberán ser removidos de DefinitelyTyped para evitar que generen confusión.
+
+Se puede remover ejecutando `npm run not-needed -- typingsPackageName asOfVersion sourceRepoURL [libraryName]`.
+- `typingsPackageName`: Este es el nombre del directorio que tienes que eliminar.
+- `asOfVersion`: Un stub será publicado a `@types/foo` con esta versión. Debería ser mas grande que cualquier versión publicada actualmente.
+- `sourceRepoURL`: Esto debería señalar el repositorio que contiene los typings.
+- `libraryName`: Un nombre descriptivo de la biblioteca, p.ej. "Angular 2" en vez de "angular2". (Si es omitido, será identico a "typingsPackageName".)
+
+Cualquier otro paquete en DefinitelyTyped que referencia el paquete eliminado deberá ser actualizado para referenciar los tipos de bundled. para hacer esto, añade `package.json` con `"dependencies": { "foo": "x.y.z" }`.
```

Comment:
Change to:
Cualquier otro paquete en DefinitelyTyped que referencie el paquete eliminado deberá ser actualizado para referenciar los tipos bundled. Para hacer esto, añade `package.json` con `"dependencies": { "foo": "x.y.z" }`.